terragrunt/aws/api/iam.tf

```diff
@@ -129,6 +129,16 @@ data "aws_iam_policy_document" "api_policies" {
       aws_ssm_parameter.api_auth_token.arn
     ]
   }
+
+  statement {
+    effect = "Allow"
+    actions = [
+      "sts:AssumeRole",
+    ]
+    resources = [
+      "arn:aws:ssm:${var.region}:*:role/ScanFilesGetObjects"
```

Comment:
typo, you have ssm instead of iam.

Reply:
good catch - that would've been super annoying to troubleshoot